.github/workflows/test_in_devenv.yml

```diff
@@ -120,13 +120,21 @@ jobs:
           run: |
             cd $CUDAQ_REPO_ROOT
             pip install . --user -vvv
-            python3 -m pytest python/tests/
+            python3 -m pytest --ignore python/tests/backends python/tests/
             pytest_status=$?
             if [ ! $pytest_status -eq 0 ]; then
               echo "pytest status = " $pytest_status
               exit 1
-            fi 
-
+            fi
+            for backendTest in python/tests/backends/*.py; do
```

Comment:
Is this to force a sequential execution?

Reply:
Yes, for some reason I ended up with issues trying to let pytest run them all in parallel.